test/DocumentFormat.OpenXml.Tests/Spreadsheet/CellValueTests.cs

```diff
@@ -180,6 +205,26 @@ public void CellDecimalTestNegative(string input)
             Assert.False(value.TryGetDecimal(out _));
         }
 
+        [Fact]
+        public void CellDecimalCultureTest()
+        {
+            // Change current culture
+            CultureInfo culture, oldculture;
```

Comment:
Why not use the new helper?

Reply:
just hadn't applied it, was waiting for your confirmation on the approach. it's applied now with your suggestions.